app/models/in_progress_form.rb

```diff
@@ -91,8 +92,45 @@ def expires_after
       end
   end
 
+  def log_cfi_metric(params)
+    return unless form_id == '21-526EZ'
+
+    # if the cfiMetric is present in the form already, then the disability education has been seen
+    if metadata['cfiMetric'].present?
+      params[:metadata]['cfiMetric'] = true
```

Comment:
Is there a reason to re-set this to true? When present, it would only ever be true, right?